indico/modules/events/registration/util.py

```diff
@@ -577,3 +581,14 @@ def import_registrations_from_csv(regform, fileobj, skip_moderation=True, notify
         })
     return [create_registration(regform, data, notify_user=notify_users, skip_moderation=skip_moderation)
             for data in todo]
+
+
+def get_registered_contribution_participants(event_id):
+    """Get user ids of registered users who participate in contributions for specific event."""
```

Comment:
While thinking about a different working for this docstring I just realized that the contribution joining isn't really needed here... so this whole util could probably be replaced with something like this:

```python
query = event.registrations.with_entities(Registration.user_id).filter(Registration.user_id.isnot(None)
return {id_ for id_, in query)
```

Might still be nice to keep it as a util here and name it something like `get_registered_user_ids`.